### PR TITLE
kh360-react-592-hardcoded-size-in-features

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
Refactor hardcoded sizes - features
[#592](https://github.com/nerubia/kh360-react/issues/592)

### CODE REVIEW

File: src/features/admin/evaluation-administrations/[id]/progress/evaluation-progress-list.tsx [hunk 1]
```diff
@@ -373,14 +373,14 @@ export const EvaluationProgressList = () => {
                   )}
               </div>
               {evaluatorToggledState[evaluatorIndex] && (
-                <table className='w-[90%] ml-14 mb-5 table-fixed whitespace-normal'>
+                <table className='w-9/10 ml-14 mb-5 table-fixed whitespace-normal'>
```
Comment (emmaacha) [inline src/features/admin/evaluation-administrations/[id]/progress/evaluation-progress-list.tsx:376]: I think these were not added in the tailwind config file?

**Review by emmaacha — CHANGES_REQUESTED** [review]

File: src/features/admin/evaluation-administrations/[id]/progress/evaluation-progress-list.tsx (hunk 1 above)
Comment (chadiegilnerubia) [reply src/features/admin/evaluation-administrations/[id]/progress/evaluation-progress-list.tsx:376]: Donee :3

File: src/pages/sample/sample.tsx [hunk 2]
```diff
@@ -736,7 +736,7 @@ export default function Sample() {
               </button>
             </div>
             <div className='flex-1 flex flex-col justify-center items-center gap-4 p-5'>
-              <div className='w-20 h-20 bg-black rounded-full'></div>
+              <div className='w-5 h-5 bg-black rounded-full'></div>
```
Comment (emmaacha) [inline src/pages/sample/sample.tsx:739]: I think this shouldn't be changed?

File: src/features/admin/evaluation-administrations/[id]/preview/preview-employees-table.tsx [hunk 3]
```diff
@@ -25,7 +25,7 @@ export const PreviewEmployeesTable = () => {
           <table className='w-full table-fixed'>
             <thead className='sticky top-0 bg-white text-left'>
               <tr>
-                <th className='w-20'></th>
+                <th className='w-5'></th>
```
Comment (emmaacha) [inline src/features/admin/evaluation-administrations/[id]/preview/preview-employees-table.tsx:28]: I think this shouldn't be changed?

File: src/features/admin/evaluation-administrations/[id]/preview/preview-employees-table.tsx [hunk 4]
```diff
@@ -81,7 +81,7 @@ export const PreviewEmployeesTable = () => {
                 {allUsers.map((employee: User) =>
                   !selectedEmployeeIds.includes(employee.id) ? (
                     <tr key={employee.id}>
-                      <td className='w-20'></td>
+                      <td className='w-5'></td>
```
Comment (emmaacha) [inline src/features/admin/evaluation-administrations/[id]/preview/preview-employees-table.tsx:84]: I think this shouldn't be changed?

File: src/features/evaluations/[id]/evaluations-list.tsx [hunk 5]
```diff
@@ -180,7 +180,7 @@ export const EvaluationsList = () => {
                     <div
                       className={`flex-1 flex flex-col text-start ${
                         evaluation.status === EvaluationStatus.Open ? "font-bold" : ""
-                      } w-20 h-15`}
+                      } w-5 h-15`}
```
Comment (emmaacha) [inline src/features/evaluations/[id]/evaluations-list.tsx:183]: I think this shouldn't be changed?

**Review by emmaacha — CHANGES_REQUESTED** [review]

File: src/pages/sample/sample.tsx (hunk 2 above)
Comment (chadiegilnerubia) [reply src/pages/sample/sample.tsx:739]: Previously it was defined as w-[20px], but I modified it to w-20. Later, I discovered that Tailwind includes a built-in w-5 utility, which achieves the same width as w-20px.

**Review by gborgonia-ir — APPROVED** [review]

**Review by emmaacha — APPROVED** [review]

**Pull request merged**